texar/modules/pretrained/__init__.py

```diff
@@ -19,3 +19,6 @@
 from texar.modules.pretrained.bert_utils import *
 from texar.modules.pretrained.gpt2_base import *
 from texar.modules.pretrained.gpt2_utils import *
+from texar.modules.pretrained.xlnet_base import *
+from texar.modules.pretrained.xlnet_model_utils import *
```

Comment:
Do we need to export `xlnet_model_utils`? I think these are modules used only in XLNet.